### PR TITLE
feat(css/parser): implement <general-enclosed> for `@media` at-rules

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Ref: https://www.w3.org/TR/mediaqueries-5/#typedef-general-enclosed

**BREAKING CHANGE:**

Yes

**Related issue (if exists):**

No

### CODE REVIEW

**Comment by alexander-akait** [comment]
@kdy1 Ready for review

File: crates/swc_css_codegen/tests/fixture/at-rules/container/input.css [hunk 1]
```diff
@@ -77,7 +77,7 @@ main, aside {
     }
 }
 
-@container (100 < width > 150px) {
+@container (100px > width > 150px) {
```
Comment (alexander-akait) [inline crates/swc_css_codegen/tests/fixture/at-rules/container/input.css:80]: It was typo

**Comment by kdy1** [comment]
CI of swc_html_minifier failed

**Comment by alexander-akait** [comment]
Yeah, WIP on this

**Comment by alexander-akait** [comment]
Fixed

**Review by kdy1 — APPROVED** [review]
Thanks!

---

swc-bump:
 - swc_css_ast --breaking

**Pull request merged**